frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/stretch-cluster.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  FormGroup,
+  Alert,
+  Checkbox,
+  Select,
+  SelectVariant,
+  SelectOption,
+  TextContent,
+  TextVariants,
+  Text,
+  SelectProps,
+} from '@patternfly/react-core';
+import { FieldLevelHelp } from '@console/internal/components/utils';
+import {
+  createNodeZoneMap,
+  isValidStretchClusterTopology,
+  ZoneNodeMap,
+} from '../../../../utils/create-storage-system';
+import { arbiterText } from '../../../../constants';
+import { WizardNodeState, WizardState } from '../../reducer';
+import { AdvancedSubscription } from '../../advanced-subscription/advanced-subscription';
+
+const EnableArbiterLabel: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="ocs-enable-arbiter-label">
+      <span className="ocs-enable-arbiter-label__title--padding">
+        {t('ceph-storage-plugin~Enable arbiter')}
+      </span>
+      <AdvancedSubscription />
+    </div>
+  );
+};
```

Comment:
Export https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/capacity-and-nodes.tsx#L73-L84
and use it here instead.